frontend/src/logging/logger.js

```diff
@@ -1,25 +1,26 @@
 import axios from "axios";
 import { createLogger } from "redux-logger";
 import { store } from "./../index";
-const {
-  NODE_ENV,
-  REACT_APP_LOGGING,
-  REACT_APP_LOG_LEVEL,
-  REACT_APP_LOGGING_SERVICE_HOST,
-  REACT_APP_LOGGING_SERVICE_PORT,
-  REACT_APP_LOGGING_SERVICE_HOST_SSL,
-  REACT_APP_LOGGING_PUSH_INTERVAL
-} = process.env;
+import config from "./../config";
+
+// Create config variables
+const isLoggingEnabled = config.logging.isEnabled;
+const logLevel = config.logging.logLevel;
+const envMode = config.envMode;
+const isLoggingHostSSL = config.logging.isHostSSL;
+const loggingSvcHost = config.logging.serviceHost;
+const loggingSvcPort = config.logging.servicePort;
+const loggingPushInterval = config.logging.pushInterval;
```

Comment:
here I would also not create variables but use the config object directly. Or if you prefer just create one variable `const loggingConfig = config.logging`